studygroups/signals.py

```diff
@@ -64,6 +64,16 @@ def handle_new_application(sender, instance, created, **kwargs):
     welcome_message.attach_alternative(learner_signup_html, 'text/html')
     welcome_message.send()
 
+@receiver(pre_save, sender=StudyGroup)
```

Comment:
@s-kennedy I'm moving this to the StudyGroup model in an overridden save model, it keeps the logic a little less fragmented.